database/seeders/PostLayoutTableSeeder.php

```diff
@@ -13,10 +13,19 @@ class PostLayoutTableSeeder extends Seeder
      */
     public function run(): void
     {
-        $layouts = ['abilities', 'assets', 'attributes', 'connection_map', 'inventory'];
+        $layouts = ['abilities', 'assets', 'attributes', 'connection_map', 'inventory', 'character_orgs', 'quest_elements', 'location_characters'];
         foreach ($layouts as $layout) {
+            $entity_type = null;
+            if ($layout == 'character_orgs') {
+                $entity_type = config('entities.ids.character');
+            } elseif ($layout == 'quest_elements') {
+                $entity_type = config('entities.ids.quest');
+            } elseif ($layout == 'location_characters') {
+                $entity_type = config('entities.ids.location');
+            }
             DB::table('post_layouts')->insert([
```

Comment:
Check your database. I have a lot of duplicates created each time I run `sail artisan db:seed`, which Kanka prod does with every release